doc/minetest.6

```diff
@@ -119,7 +119,7 @@ Display an interactive terminal over ncurses during execution.
 
 .SH ENVIRONMENT
 .TP
-.B MINETEST_SUBGAME_PATH
+.B MINETEST_GAME_PATH
```

Comment:
`test_servermodmanager.cpp` also needs renaming.

Reply:
I renamed it, but probably both `MINETEST_GAME_PATH` and `MINETEST_SUBGAME_PATH` should be tested at some point.